packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -148,6 +156,8 @@ contract SystemConfig is OwnableUpgradeable, ISemver, IGasToken {
             _blobbasefeeScalar: 0,
             _batcherHash: bytes32(0),
             _gasLimit: 1,
+            _eip1559Denominator: 1,
```

Comment:
We agreed on not modifying the initializer with inputs to reduce the integration pains with OP Stack Chain Manager. We agreed on emitting default canyon values, if the storage slots are set to 0 (first time the contract is initalized) rather than having input arguments. This will sync the L1 state with the L2 view of the state. The dependency that is introduced is that chains will need to update to a newer release of `op-node` BEFORE we do the contract upgrade. We will add a getter/setter for the values, which can be atomically called during the same batch the new implementation is set in the proxy. cc @roberto-bayardo for context. Probably easier to just start on a fresh branch